### ISSUE TITLE
Templater system suggester “Popup” is not well readable

### ISSUE THREAD

**Issue by Buddinski88** [aaaaalexis/obsidian-cupertino#8]
Hi Aaaaalexis,

thanks in advance for the great theme. So far I really like it and it makes Obsidian much clearer. Keep it up :-)

I use the function tp.system.suggester for my diary, which opens a popup when creating the diary entry in which I can select the respective values.
Currently there are 11 queries/properties and when you see the popup, it's not really easy to read. Example:
![image](https://github.com/user-attachments/assets/930c1c63-6bbf-49e7-a59a-2bad3cfc1764)

This is the part of my template that is called when it is created:

```markdown

> [!Do]+ Habit Tracker  
> Morning_routine:: <%tp.system.suggester(["🟩️","🟥️"],["1","0"], false, "Morning routine done 🌅?")%>
> Exercise:: <%tp.system.suggester(["🟩️","🟥️"],["1","0"], false, "Did i some 🏋️‍♀️?")%>
> Reading:: <%tp.system.suggester(["🟩️","🟥️"],["1","0"], false, "Did i 📖 something today?")%> 
> Writing:: <%tp.system.suggester(["🟩️","🟥️"],["1","0"], false, "Did i 📝 about something?")%>
> Fresh_air:: <%tp.system.suggester(["🟩️","🟥️"],["1","0"], false, "Did i got some Fresh air 💨?")%>
> Paperwork:: <%tp.system.suggester(["🟩️","🟥️"],["1","0"], false, "Did I do any 🗳️ today ?")%>
> Quality_time_with_Katrin:: <%tp.system.suggester(["🟩️","🟥️"],["1","0"], false, "Did I have quality time with 👸?")%> 
> Quality_time_with_Kids:: <%tp.system.suggester(["🟩️","🟥️"],["1","0"], false, "Did I have quality time with the 🤹?")%>
> Media_detox:: <%tp.system.suggester(["🟩️","🟥️"],["1","0"], false, "Have I spent less than 2 hours on my 📱?")%>
> Headache:: <%tp.system.suggester(["🟪️","🟦️","🟩️","🟨️","🟧️","🟥️"],["0","1","2","3","4","5"], false, "What about 🤯?")%>
> Mood:: <%tp.system.suggester(["🟪️","🟦️","🟩️","🟨️","🟧️","🟥️"],["5","4","3","2","1","0"], false, "How 👍 was my mood today?")%> 

```

Can you correct this so that it is easier to read?

Many thanks and best regards,
Buddinski88

**Comment by aaaaalexis** [comment]
Is `Morning_routine` or `Mood` that's supposed to be the first one that you fill in?

**Comment by Buddinski88** [comment]
> Is it `Morning_routine` or `Mood` that's supposed to be the first one that you fill in?

`Mood` is the first item and `Morning_routine` the last.

**Comment by aaaaalexis** [comment]
Fixed in 1.1.2 https://github.com/aaaaalexis/obsidian-cupertino/commit/6addce71e0aebd9a8987d2a109f7b3d3b4dee14b
